repos/spack_repo/builtin/packages/motioncor2/package.py

```diff
@@ -19,46 +17,76 @@ class Motioncor2(Package):
     to keep up with automated data collection."""
 
     homepage = "http://msg.ucsf.edu/em/software"
-    manual_download = True
+    # Use Scipion's download server as the official site no longer hosts some deprecated versions
+    url = "https://scipion.cnb.csic.es/downloads/scipion/software/em/motioncor2-1.6.4.tgz"
 
-    version("1.6.4", sha256="28bb3e6477abf34fe41a78bcb9da9d77d08e2e89ecd41240fab085a308e6c498")
-    version("1.4.7", sha256="8c33969b10916835b55f14f3c370f67ebe5c4b2a9df9ec487c5251710f038e6b")
+    version("1.6.4", sha256="bcb70ddde38daa7722e007f2509bae928c8603530524b6a36ed79f4ca7b3edb8")
+    version("1.5.0", sha256="20cdb46efe3f3c578071ddbd9e9f45c8462b0cbb2fc5b3cfb2d0b66ded0d30e0")
+    version("1.4.5", sha256="2f04bceb6069cd4f91a51447aac07b887be46206e094555bc445e89944ed2851")
 
-    # None of the below are available for download
-    version(
-        "1.1.0",
-        sha256="6e37e7ed63a9f0aab5d794b2604d5ba79333960bb9440a1a218630b03dbeaeac",
-        deprecated=True,
-    )
-    version(
-        "1.0.5",
-        sha256="4efa55af25644bcff1ca7882419267b8c094c9cc6155b37d2c204b154c56f5a8",
-        url="http://msg.ucsf.edu/MotionCor2/MotionCor2-1.0.5.tar.gz",
-        deprecated=True,
-    )
-    version(
-        "1.0.4",
-        sha256="c75738160ac18d3f27c33677e78e63313d8ec2b023b5a46173428c3fa0451a94",
-        url="http://msg.ucsf.edu/MotionCor2/MotionCor2-1.0.4.tar.gz",
-        deprecated=True,
-    )
+    # Deprecated versions, no longer available in official site
+    with default_args(deprecated=True):
+        version("1.6.3", sha256="16c97516992d638ea817a943185213874dfad7d87a6bb71e102598922bb35c75")
```

Comment:
As said, in genereal and despite breaking changes in minor versions like you said, there should be no reason to add new versions (like 1.6.3, AFAICS) as deprecated, unless you know a specific case were someone needs that version.
```suggestion
```